server/lib/field_hub/couch_service.ex

```diff
@@ -337,6 +339,85 @@ defmodule FieldHub.CouchService do
     )
   end
 
+  @doc """
+  Returns up to n last changes for the specified project.
+  If the project has less than n changes or is new, less or no change would be returned.
+
+  __Parameters__
+  - `project_identifier` - The name of the project.
+
+  ## Example
+  iex> get_n_last_changes("project_a",1)
+  [
+    %{
+      "changes" => [%{"rev" => "2-4f773e67d44d4bc99008713d9f9d164f"}],
+      "doc" => %{
+        "_id" => "988d0107-952f-4eaf-8c1f-4a25a0c5b15c",
+        "_rev" => "2-4f773e67d44d4bc99008713d9f9d164f",
+        "created" => %{
+          "date" => "2023-10-17T14:56:31.120Z",
+          "user" => "anonymous"
+        },
+        "modified" => [
+          %{"date" => "2023-10-17T14:57:55.929Z", "user" => "anonymous"}
+        ],
+        "resource" => %{
+          "category" => "Survey",
+          "id" => "988d0107-952f-4eaf-8c1f-4a25a0c5b15c",
+          "identifier" => "area 190",
+          "relations" => %{}
+        }
+      },
+      "id" => "988d0107-952f-4eaf-8c1f-4a25a0c5b15c",
+      "seq" => "6-g1AAAACLeJzLYWBgYMpgTmHgzcvPy09JdcjLz8gvLskBCScyJNX___8_K4M5kTkXKMBuYW5iaZpkgK4Yh_Y8FiDJ0ACk_qOaYpJmlpSSiq4nCwA51yqW"
+    }
+  ]
+
+  """
+  def get_n_last_changes(project_identifier, n) do
```

Comment:
There should probably a guard concerning `n` (not more than 100).

Reply:
function name changed for get_last_n_changes().
Limited to 100 last changes. 
The slider range is 2-100, I set the minimum at 2  because of the s in the text, to avoid: "Show the 1 last changes" , or  "Show the 0 last changes"